hathor/manager.py

```diff
@@ -1094,6 +1127,21 @@ def is_healthy(self) -> Tuple[bool, Optional[str]]:
 
         return True, None
 
+    def check_sync_state(self):
+        now = time.time()
+
+        if self.has_recent_activity():
+            self.first_time_fully_synced = now
+
+            total_sync_time = LogDuration(self.first_time_fully_synced - self.start_time)
+            vertex_count = self.tx_storage.get_tx_count() + self.tx_storage.get_block_count()
```

Comment:
Maybe use `tx_storage.get_count_tx_blocks()` here as well?

Reply:
Actually I was doing the same thing as here on the other place where I needed this.

Do you think it's better to use `tx_storage.get_count_tx_blocks()` on both instead?

Reply:
I agree we should use the same method in both cases. Let's discuss which one is better there.